src/components/about/description/AboutDescription.module.css

```
@@ -0,0 +1,18 @@
+.box {
+  flex: 1 1 auto;
+  height: 100vh;
```

You have used 100vh for the main container already. 
here I suggest to dance from the parent size
```suggestion
  height: 100%;
```

---

Fixed